python_modules/dagster/dagster/_core/storage/captured_log_manager.py

```diff
@@ -127,12 +129,12 @@ def fetch(self):
                 self._cursor = log_data.cursor
             should_fetch = _has_max_data(log_data.stdout) or _has_max_data(log_data.stderr)
 
-    def complete(self):
+    def complete(self) -> None:
         self.is_complete = True
 
 
-def _has_max_data(chunk) -> bool:
-    return chunk and len(chunk) >= MAX_BYTES_CHUNK_READ
+def _has_max_data(chunk: Optional[bytes]) -> bool:
+    return chunk and len(chunk) >= MAX_BYTES_CHUNK_READ  # type: ignore
```

Comment:
why `type: ignore`? Seems like it should work

Reply:
Function is used as a predicate but does not actually return a `bool` (if `chunk` is truthy). Will be fixed in a `[typing/runtime]` upstack.